tests/headers/use-core.h

```diff
@@ -0,0 +1,6 @@
+// bindgen-flags: --use-core --raw-line "extern crate core;"
```

Comment:
Shouldn't `--use-core` imply `extern crate core;` in the bindings? Shouldn't we automatically insert the extern crate definition?